ee/modules/030-cloud-provider-openstack/templates/namespace.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 kind: Namespace
 metadata:
   name: d8-cloud-provider-openstack
-  {{- include "helm_lib_module_labels" (list .) | nindent 2 }}
+  {{- include "helm_lib_module_labels" (list . (dict "prometheus.deckhouse.io/rules-watcher-enabled" "")) | nindent 2 }}
```

Comment:
We need to add `true` value for label `prometheus.deckhouse.io/rules-watcher-enabled`.

You skipped it for many times.